crates/core/src/vfs.rs

```diff
@@ -530,19 +510,16 @@ impl OpenFile {
     pub fn read_at<P, S: IndexedFull>(
         &self,
         repo: &Repository<P, S>,
-        mut offset: usize,
+        offset: usize,
         mut length: usize,
     ) -> RusticResult<Bytes> {
-        // find the start of relevant blobs => find the largest index such that self.content[i].starts_at <= offset, but
-        // self.content[i+1] > offset  (note that a last dummy element has been added)
-        let mut i = self.content.partition_point(|c| c.starts_at <= offset) - 1;
-
-        offset -= self.content[i].starts_at;
+        let (mut i, mut offset) = self.offsets.offset(offset);
```

Comment:
`let (mut i, mut offset) = self.offsets.offset(offset);` that looks like a code smell imho, as we are using four times the same term in a single line.

Maybe we can make variables or methods/functions more self-descriptive here.

Reply:
done.